usergroups.go

```diff
@@ -121,16 +122,44 @@ func (api *Client) EnableUserGroupContext(ctx context.Context, userGroup string)
 	return response.UserGroup, nil
 }
 
+// GetUserGroupsParams contains arguments for GetUserGroups call
+type GetUserGroupsParams struct {
+	// IncludeCount	- Include the number of users in each User Group. Default: false
+	IncludeCount bool
+	// IncludeDisabled - Include disabled User Groups. Default: false
+	IncludeDisabled bool
+	// IncludeUsers - Include the list of users for each User Group. Default: false
+	IncludeUsers bool
+}
+
+// NewGetUserGroupsParams creates a new instance GetUserGroupsParams with defaults
+func NewGetUserGroupsParams() GetUserGroupsParams {
```

Comment:
this is kind of pointless, zero value is already false. this entire function can be dropped and replaced with `GetUserGroupsParams{}`